easybuild/easyconfigs/a/AlphaFold/AlphaFold-2.1.1-foss-2021a-CUDA-11.3.1.eb

```diff
@@ -0,0 +1,192 @@
+easyblock = 'PythonBundle'
+
+name = 'AlphaFold'
+version = '2.1.1'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://deepmind.com/research/case-studies/alphafold'
+description = "AlphaFold can predict protein structures with atomic accuracy even where no similar structure is known"
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+builddependencies = [
+    # CMake/Doxygen/SWIG are required for building OpenMM
+    ('CMake', '3.20.1'),
+    ('Doxygen', '1.9.1'),
+    # required for installing dm-tree
+    ('Bazel', '3.7.2'),
+    ('SWIG', '4.0.2'),
+]
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('SciPy-bundle', '2021.05'),
+    ('PyYAML', '5.4.1'),
+    ('TensorFlow', '2.6.0', versionsuffix),
+    ('Biopython', '1.79'),
+    ('HH-suite', '3.3.0'),
+    ('HMMER', '3.3.2'),
+    ('Kalign', '3.3.1'),
+    ('jax', '0.2.24', versionsuffix),  # also provides absl-py
+    ('CUDA', '11.3.1', '', True),
```

Comment:
Shouldn't we consistently include UCX-CUDA whenever we use CUDA on something that can do MPI?
And I would put CUDA before anything using versionsuffix becuse it looks better that way.

Reply:
Yes. Done in e6570c2